Example/Tests/BPKShadowTest.m

```diff
@@ -0,0 +1,39 @@
+//
+//  BPKShadowTest.m
+//  Backpack_Tests
+//
+//  Created by Hugo Tunius on 08/02/2018.
+//  Copyright © 2018 Hugo Tunius. All rights reserved.
+//
+
+#import <XCTest/XCTest.h>
+
+#import <Backpack/Shadow.h>
+
+
+@interface BPKShadowTest : XCTestCase
+@property(nonatomic, copy) NSArray<NSString *> *selectors;
+@end
+
+@implementation BPKShadowTest
+
+- (void)setUp {
+    self.selectors =  @[ @"shadowSm", @"shadowLg", @"shadowXl" ];
+}
+
+- (void)testExpectedShadowStylesAreAvailable {
+    for (NSString *selectorString in self.selectors) {
+        SEL selector = NSSelectorFromString(selectorString);
+        XCTAssert([BPKShadow respondsToSelector:selector]);
+        XCTAssertNotNil([BPKShadow performSelector:selector]);
+    }
+}
+
+- (void)testStylesAreCached {
+    BPKShadow *shadow1 = [BPKShadow shadowSm];
+    BPKShadow *shadow2 = [BPKShadow shadowSm];
+
+    XCTAssertTrue(shadow1 == shadow2, @"BPKShadow should cache shadow styles");
```

Comment:
This verifies we never create more than a single value per shadow type by doing pointer comparison. Object comparison in objective-c is done with `isEqual:`